### PR TITLE
Let's reiterate Linux gcc warnings

### PR DESCRIPTION
**Pull request by singalen**
Not all of them though.
Is it OK to re-enable Freetype ans libssh?

### CODE REVIEW

File: src/fileopers.cpp [hunk 1]
```diff
@@ -1343,7 +1343,7 @@ bool OperCFThread::CopyFile( FS* srcFs, FSPath& srcPath, FSNode* srcNode, FS* de
 	out = destFs->OpenCreate( destPath, false | commitAll, srcNode->st.mode, 0, &ret_err, Info() );
 
 	if ( out < 0 && destFs->IsEEXIST( ret_err ) )
```
Comment (viktor-podzigun) [inline src/fileopers.cpp]: We may also fix spelling bellow: Overwrite

File: src/fileopers.cpp [hunk 2]
```diff
@@ -1343,7 +1343,7 @@ bool OperCFThread::CopyFile( FS* srcFs, FSPath& srcPath, FSNode* srcNode, FS* de
...
-		switch ( RedMessage( _LT( "Owerwrite  file?\n" ) , destFs->Uri( destPath ).GetUtf8(), bOkAllNoCancel ) )
+		switch ( RedMessage( _LT( "Owerwrite file?\n" ) , destFs->Uri( destPath ).GetUtf8(), bOkAllNoCancel ) )
```
Comment (corporateshark) [inline src/fileopers.cpp]: Please, check the localization files and fix them accordingly.

**Comment by corporateshark** [comment]
Enabling FreeType and libssh is ok.

File: src/fileopers.cpp (hunk 1 above)
Comment (singalen) [inline src/fileopers.cpp]: Done!

File: src/fileopers.cpp (hunk 2 above)
Comment (singalen) [inline src/fileopers.cpp]: Done.

**Pull request merged**